apps/api/src/schema/library-traffic.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from "@hono/zod-openapi";
+
+export const libraryTrafficQuerySchema = z.object({
+  libraryName: z
+    .string()
+    .openapi({
+      example: "Langson Library",
+      description: "Filter results by library name",
+    })
+    .optional(),
+  locationName: z
+    .string()
+    .openapi({
+      example: "4th Floor - Nordstrom Honors Study Room",
+      description: "Filter results by exact location name",
+    })
+    .optional(),
+});
+
+export const libraryTrafficEntrySchema = z.object({
+  id: z.number().int().nonnegative().openapi({ example: 245 }),
+  libraryName: z.string().openapi({
+    example: "Langson Library",
+    description: "Name of the library that owns this location",
```

Comment:
"Owns" is unusual language that may raise some mental questions - I think you should reword using something like "the library which contains"